tab_automl/automl/datasets.py

```diff
@@ -161,3 +190,37 @@ def prepare_x_and_y(self,
         print("X feature set and target feature has been split...")
 
         return x, y
+    
+class Credit_Card_Customer_Data:
+    """
+
+    Credit_Card_Customer_Data Dataset
+   ============
+
+    This class will be an example for
+    other clustering datasets or for
+    custom datasets prepared on comma
+    separated value format to be precise.
+
+    """
+
+    def __init__(self):
+        # reading the dataset from source
+        self.data = pd.read_csv("tab_automl/datasets/Credit_Card_Customer_Data.csv")
+        # storing the columns overview
+        self.columns = pd.Series([str(self.data[feature].dtype) for feature in self.data.columns])
+        self.labels = None
+        print(f"Populated the dataframe with data records...")
+
+    def prepare_x(self,
```

Comment:
In clustering, as all the columns will be used for clustering, there's no need to add `feature_set_columns`.